ReactNative/ReactTwitter/core/service/percent-encoder.js

```diff
@@ -0,0 +1,7 @@
+class PercentEncoder {
```

Comment:
I used the terminology from the twitter dev docs here: https://dev.twitter.com/oauth/overview/percent-encoding-parameters